integrationTests/consensus/testInitializer.go

```diff
@@ -245,7 +245,7 @@ func createConsensusOnlyNode(
 	pubKeys []crypto.PublicKey,
 	testKeyGen crypto.KeyGenerator,
 	consensusType string,
-	epochStartRegistrationHandler mainFactory.EpochStartNotifier,
+	epochStartRegistrationHandler factory2.EpochStartNotifier,
```

Comment:
might rename this or rename the "other factory" involved

Reply:
done